src/stylesheets/elements/_list.scss

```diff
@@ -27,7 +27,9 @@
 
 // Unstyled lists
 @include override-prose {
-  .usa-unstyled-list {
+  .usa-list--unstyled {
     @include unstyled-list;
   }
 }
+
+// TODO: check lists to see if line height is working ok
```

Comment:
Has this been checked? Is the comment meant to stay?

Reply:
Whoops, yeah, we checked and it's ok